Duplicati/License/Duplicati.License.csproj

```diff
@@ -70,6 +70,15 @@
       <Link>licenses\gpg\License.txt</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="..\..\thirdparty\Otp.NET\Homepage.txt">
+      <Link>licenses\Otp.NET\Homepage.txt</Link>
+    </Content>
+    <Content Include="..\..\thirdparty\Otp.NET\license.txt">
+      <Link>licenses\Otp.NET\license.txt</Link>
+    </Content>
+    <Content Include="..\..\thirdparty\Otp.NET\licensedata.json">
+      <Link>licenses\Otp.NET\licensedata.json</Link>

```

Comment:
Each of these needs the `<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>` element.

Reply:
My bad, it's fixed!